config/config.go

```diff
@@ -124,7 +124,7 @@ var (
 			EnableStakingIndexer:          false,
 			CompressBlock:                 false,
 			AllowedBlockGasResidue:        10000,
-			MaxCacheSize:                  0,
+			StateMaxCacheSize:             1000,
```

Comment:
we will need some tests to measure statedb read time, throughput, memory usage, etc. under different values of cache size so we can get a proper default value

Reply:
addressed on latest commit!